praw/__init__.py

```diff
@@ -238,6 +238,13 @@ def config_boolean(item):
         self.https_proxy = (obj.get('https_proxy') or
                             os.getenv('https_proxy') or None)
         # We use `get(...) or None` because `get` may return an empty string
+
```

Comment:
All settings are inherited from the default section of praw.ini. Setting it to true there makes it the default.